docs/index.rst

```diff
@@ -40,6 +39,16 @@ encoding
   Encoding to be used to read an OpenAPI spec. If not passed, Sphinx's
   source encoding will be used.
 
+paths
+  A comma separated list of paths to filter the included openapi spec by. For example:
+
+  .. code:: restructuredtext
+
+      .. openapi:: specs/openapi.yml
+          :paths: /persons,/evidence
```

Comment:
Python has a built-in string function to split by newlines, I used that rather than a custom regex.